.github/workflows/nightly-throughput-stress.yml

```diff
@@ -130,13 +138,37 @@ jobs:
             --duration $TEST_DURATION \
             --timeout $TEST_TIMEOUT \
             --max-concurrent 10 \
+            --prom-listen-address 127.0.0.1:9091 \
+            --worker-prom-listen-address 127.0.0.1:9092 \
+            --prom-instance-addr 127.0.0.1:9090 \
+            --prom-instance-config \
+            --prom-export-worker-metrics $RUN_ID.parquet \
             --option internal-iterations=10 \
             --option continue-as-new-after-iterations=3 \
             --option sleep-time=1s \
             --option visibility-count-timeout=5m \
             --option min-throughput-per-hour=1000 \
             2>&1 | tee $WORKER_LOG_DIR/scenario.log
 
+      - name: Configure AWS credentials
```

Comment:
Are there no secrets required here? So anyone can upload to this bucket?

Reply:
We assume an ARN that gives us perms to write to this specific bucket, from specific github repos (the sdk repos)

Reply:
From what I was told, the ARN only permits uploads from our SDK repos. (based on its trust policy), but I can move it behind a secret regardless

Reply:
I'd be interested to confirm how that works. So the ARN restricts to uploading from GH worker IPs only? Yes, would definitely prefer using secrets when uploading.